scm/src/gmtb_scm_type_defs.f90

```diff
@@ -1679,6 +1847,16 @@ subroutine physics_associate(physics, scm_state, col)
       physics%Sfcprop(col)%spec_lh_flux => scm_state%lh_flux
       physics%Sfcprop(col)%zorl => scm_state%sfc_roughness_length_cm
     endif
+    if(scm_state%model_ics) then
```

Comment:
I wonder why all surface-related variables aren't wrapped in the model_ics test?

Reply:
These surface fields were already there, so I did not want to break existing functionality, and I the surface_roughness_lenght is also used by the MP and PBL schemes.